van-gogh-second-page/jons/index-tickets.css

```diff
@@ -0,0 +1,125 @@
+@import url('https://fonts.googleapis.com/css2?family=Varela+Round&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@100;400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Josefin+Sans&display=swap');
```

Comment:
Same here

Reply:
I use "Varela" but I´ve removed the others